test/packages/good/_dev/build/build.yml

```diff
@@ -0,0 +1,3 @@
+dependencies:
```

Comment:
Just approved the other PR but we have one more name here: `dependency`.

I assume the ecs dependency can only be used once, should we add it by name instead of array?

```
dependencies:
  ecs:
    reference: foo
```

Reply:
Replaced array with objects. I assume it's just to prevent custom dependencies that users may try to introduce and we don't want them now.